save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/projects/ProjectProblemView.kt

```diff
@@ -0,0 +1,168 @@
+@file:Suppress("FILE_NAME_MATCH_CLASS")
+
+package com.saveourtool.save.frontend.components.basic.projects
+
+import com.saveourtool.save.entities.ProjectProblemDto
+import com.saveourtool.save.frontend.components.basic.cardComponent
+import com.saveourtool.save.frontend.components.basic.markdown
+import com.saveourtool.save.frontend.externals.fontawesome.*
+import com.saveourtool.save.frontend.utils.*
+import com.saveourtool.save.frontend.utils.noopLoadingHandler
+import com.saveourtool.save.validation.FrontendRoutes
+import csstype.BorderRadius
+import csstype.ClassName
+import js.core.jso
+import react.FC
+import react.Props
+import react.dom.html.ReactHTML.a
+import react.dom.html.ReactHTML.div
+import react.dom.html.ReactHTML.form
+import react.dom.html.ReactHTML.h1
+import react.dom.html.ReactHTML.nav
+import react.dom.html.ReactHTML.span
+import react.router.dom.Link
+import react.useState
+
+val projectProblem: FC<ProjectProblemViewProps> = FC {props ->
+
+    val (projectProblem, setProjectProblem) = useState(ProjectProblemDto.empty)
+
+    useRequest {
+        val projectProblemNew: ProjectProblemDto = get(
+            url = "$apiUrl/projects/problem/get/by-id?id=${props.projectProblemId}",
```

Comment:
Suggestion to use `get` that receives query params built by `jso`. You can find implementation in the same file as your `get`

Reply:
Fixed